src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -756,14 +758,17 @@ namespace winrt::TerminalApp::implementation
     // Method Description:
     // - Creates a new connection based on the profile settings
     // Arguments:
-    // - the profile GUID we want the settings from
+    // - the profile we want the settings from
     // - the terminal settings
     // Return value:
     // - the desired connection
-    TerminalConnection::ITerminalConnection TerminalPage::_CreateConnectionFromSettings(GUID profileGuid,
+    TerminalConnection::ITerminalConnection TerminalPage::_CreateConnectionFromSettings(Profile profile,
                                                                                         TerminalSettings settings)
     {
-        const auto profile = _settings.FindProfile(profileGuid);
+        if (!profile)
+        {
+            profile = _settings.ProfileDefaults();
```

Comment:
(Notes from offline discussion)
Don't forget to throw this under a feature flag. This might be a bit of a weird behavior in the stable channel because SUI won't have a way to modify `ProfileDefaults`.

Reply:
I'm changing the logic here to use the default profile until a later PR that opens up the possibility of using ProfileDefaults.